src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/services/PrisonerIndexService.kt

```diff
@@ -97,13 +99,19 @@ class PrisonerIndexService(
       }
     }
 
-    existingPrisoner?.also {
-      raiseDifferencesTelemetry(
-        offenderBooking.offenderNo,
-        offenderBooking.bookingNo,
-        getDifferencesByPropertyType(it, storedPrisoner),
-        telemetryClient
-      )
+    if (diffProperties.telemetry) {
```

Comment:
how can you assume that `diffProperties` is not null and also `telemetry` is not null?

Reply:
ignore this too 😁